lib/trino-collect/src/test/java/io/trino/collect/cache/TestEvictableCache.java

```diff
@@ -164,7 +224,7 @@ public void testInvalidateAndLoadConcurrently(Invalidation invalidation)
         int[] primes = {2, 3, 5, 7};
```

Comment:
optional: TestEvictableCache line 210

A simple test to showcase that also after the cache invalidation process is done, the cache still exposes the fresh value:
```          
 assertEquals(cache.getIfPresent(key), "fresh value");
```